---
bool success
float32[] Mean
float32[] Cov
